src/main/java/com/chimbori/crux/articles/PostprocessHelpers.java

```diff
@@ -203,4 +208,13 @@ private static void removeDisallowedAttributes(Element node) {
     }
   }
 
+  private static boolean isImportant(Node node) {
+      boolean result=false;
+      for(Node n=node;n!=null;n=n.parentNode())
+        if(n.attr("crux-important").equals("true")) {
```

Comment:
Is it necessary to check the value, instead of just checking the presence of the attribute?

Reply:
> Is it necessary to check the value, instead of just checking the presence of the attribute?

We preferred value because we have a lot of different checks managed by a lot of different people to decide which nodes are "keepers," and this allowed us to coordinate those pieces of code against different values. However, we could easily do that on another attribute, and just mark the "official" attribute at the end.

This decision is down to style, not substance. I'm happy to move to checking for presence.